src/intents.cs

```diff
@@ -7363,9 +7514,12 @@ interface INStringResolutionResult {
 		INStringResolutionResult GetConfirmationRequired (NSObject itemToConfirm, nint reason);
 	}
 
+ 	[Deprecated (PlatformName.iOS, 15, 0)]
+ 	[Deprecated (PlatformName.MacCatalyst, 15, 0)]
 	[iOS (10, 0)]
-	[Watch (3, 2)]
-	[NoTV, Mac (11,0)]
+	[NoWatch] // it says it's now NoWatch, but not deprecated? (there was previous watch availability)
```

Comment:
This is a breaking change, if the type was already present in an earlier version of Xamarin.iOS, we can't remove it now (we have to mark it as deprecated instead).

Reply:
makes sense, thanks!